src/partitioner/local/MetisPartitioner.cpp

```diff
@@ -280,6 +287,50 @@ void MetisPartitioner::createPartitionFiles(idx_t *part) {
         std::map<int, std::vector<int>> partEdgeMap = partitionedLocalGraphStorageMap[part];
         std::map<int, std::vector<int>> partMasterEdgeMap = masterGraphStorageMap[part];
 
+        std::map<long, string[7]> partitionedEdgeAttributes;
+
+
+        //edge attribute separation for partition files
+        for (auto it = partEdgeMap.begin(); it != partEdgeMap.end(); ++it) {
+            for (auto it2 = it->second.begin(); it2 != it->second.end(); ++it2) {
+                auto entry = edgeMap.find(make_pair(it->first, *it2));
+                long article_id = entry->second;
+                string attributes[7];
+                auto array = (articlesMap.find(article_id))->second;
+
+                for (int itt = 0; itt < 7; itt++) {
+                    attributes[itt] = (array)[itt];
+                }
+                for (int itt = 0; itt < 7; itt++) {
```

Comment:
Why do you have this empty for loop?

Reply:
It should be removed.

Reply:
Please fix this issue of empty for loop.